### DISCUSSION TITLE
Capturing Tables and paragraphs

### DISCUSSION THREAD

**Discussion by pratikmore96** [mindee/doctr#1276] [Q&A]
Hi,

I am trying to capture the tables and paragraphs from the 30-40 pages pdf document. I tried to use heuristics and patterns to find it out from the generated doctr Json file.  But no luck. As its very difficult to understand where the table starts or where the paragraph starts and ends by just looking at the json.

Is there any way to capture the headings, subheadings, list, tables, paragraphs, etc from the pdf document?

**Comment by felixdittrich92** [comment]
Hi @pratikmore96 👋,

docTR does not provide layout analysis or table recognition at the moment.

Best regards
